_episodes/introduction.md

```diff
@@ -28,20 +28,19 @@ those wishing to enhance the reusability of their data holdings." This guideline
 has subsequently been endorsed by working groups, funding bodies and
 institutions.
 
-The FAIR principles have a relatively strong focus on "machine-actionability".
-That means that the data should not only be *fair* for humans,
-but also for computers. This is especially relevant for data discovery and working with data.
+The FAIR principles have a strong focus on "machine-actionability".
+This means that the data should be primarily readable by computers (and not only by humans), which is particularly relevant for working and discovering new data.
 
 ![FAIR principles illustration by Scriberia](../fig/FAIRPrinciples.jpg) *This
 image was created by Scriberia for The Turing Way community and is used under a
 CC-BY licence. Source: <https://doi.org/10.5281/zenodo.3695300>*
 
-> ## FAIR is not open
+> ## FAIR is not (necessarily) Open
 >
-> Some data cannot be openly shared for example because of privacy considerations.
-> For such situations the following guideline from the EU H2020 programme may be followed:
+> Some data cannot be openly shared, for example, because of privacy considerations.
+> For such situations, the EU H2020 Program Guidelines on FAIR Data apply:
```

Comment:
```suggestion
> Some data cannot be shared openly, for example, because of privacy considerations.
> For such situations, the EU H2020 Program Guidelines on FAIR Data say that the data should be *"As open as possible, as closed as necessary."*
```